charts/kyverno/templates/_helpers.tpl

```diff
@@ -114,3 +114,11 @@ replicas: 1
 replicas: 3
 {{- end }}
 {{- end }}
+
+{{- define "kyverno.securityContext" -}}
+{{- if semverCompare "<1.19" .Capabilities.KubeVersion.Version }}
+{{ toYaml (omit .Values.securityContext "seccompProfile") }}
```

Comment:
To go along with other comment, move toYaml to template and just generate data here

Reply:
Not sure I can print an object directly in a template without `toYaml`.

Reply:
You're right, can't return non-string without a lot of extra code I believe so better to keep simple